reporthandling/results/v1/reportsummary/resourcecounters.go

```diff
@@ -9,42 +9,63 @@ type ICounters interface {
 	Passed() int
 	Skipped() int
 	Failed() int
+	Excluded() int // deprecated
 	All() int
 }
 
+type ISubCounters interface {
+	All() int
+	Ignored() int
+}
+
 // =================================== Counters ============================================
 
 // NumberOfExcluded get the number of excluded resources
-func (resourceCounters *ResourceCounters) Set(allLists *helpersv1.AllLists) {
+func (resourceCounters *StatusCounters) Set(allLists *helpersv1.AllLists) {
```

Comment:
since we renamed the struct to `StatusCounters` i suggest to rename the filename, as well as the variable names